src/django_components/slots.py

```diff
@@ -77,6 +81,7 @@ class SlotFill(NamedTuple):
     nodelist: NodeList
     context_data: Dict
     alias: Optional[AliasName]
+    scope: Optional[ScopeName]
```

Comment:
How the feature is implemented is that when we parse `slot` tags, we note down the kwargs parsed to it, and pass it down. Similarly, when we parse `fill` tags, we note down the `slot_data` variable name, and pass it down. These two then meet when we are rendering a slot, and we assign the slot kwargs to the context under a key set in `fill` tag.

`slot_data` value is required to be string literal, e.g. `"quoted_string"`, same as we do for fill name or the `as var` syntax.